src/components/ebay-breadcrumb/index.js

```diff
@@ -45,9 +45,9 @@ function getInitialState({ hijax }) {
     return { hijax };
 }
 
-function handleClick(event) {
-    eventUtils.preventDefaultIfHijax(event, this.state.hijax);
-    emitAndFire(this, 'breadcrumb-select', { el: event.target });
+function handleClick(originalEvent) {
+    eventUtils.preventDefaultIfHijax(originalEvent, this.state.hijax);
+    emitAndFire(this, 'breadcrumb-select', { originalEvent, el: event.target });
```

Comment:
i think you meant `originalEvent.target`
```emitAndFire(this, 'breadcrumb-select', { originalEvent, el: originalEvent.target });```

Reply:
Yup. I think it should be `originalEvent.target`, too.